### ISSUE TITLE
Build scroll to top button for mobile

### ISSUE THREAD

**Issue by evancp87** [TheScienceMuseum/collectionsonline#1745]
With the new child records components, with some object records potentally having hundreds or more records, some of the pages are now particularly long, especially on mobile. These pages could benefit from a scroll to top button to improve the user experience. I would imagine it would only make sense on Object records, both SPH and MPH types, rather than any others. Will need some designing, and deciding whether to build from scratch or install a 3rd party library, although it should be easy enough to build I would think.

**Comment by evancp87** [comment]
complete, with scope to improve in future from a design perspective
